app/src/test/java/com/lkand/bitcoin_tracker/feature/dashboard/DashboardViewModelUnitTest.kt

```diff
@@ -0,0 +1,59 @@
+package com.lkand.bitcoin_tracker.feature.dashboard
+
+import com.lkand.bitcoin_tracker.feature.dashboard.model.DashboardResponseModel
+import com.lkand.bitcoin_tracker.feature.dashboard.viewmodel.DashboardViewModel
+import junit.framework.Assert.assertEquals
+import org.junit.Test
+import org.junit.Rule
+import org.mockito.junit.MockitoJUnit
+import org.mockito.Mock
+import org.powermock.reflect.Whitebox
+import org.mockito.Mockito.`when` as whenever
+
+class DashboardViewModelUnitTest {
+    private val viewModel = DashboardViewModel()
+
+    @Mock
+    private lateinit var dashboardResponseModel: DashboardResponseModel
+
+    @Rule
+    @JvmField
+    val mockitoRule = MockitoJUnit.rule()
```

Comment:
`In order to prevent unexpected errors, the type should be declared explicitly`

ini bisa diubah seperti ini `val mockitoRule = MockitoJUnit.rule()!!`, untuk menghindari nullability issues.